tests/stake-pools/e2e/features/stake-pools.feature

```diff
@@ -0,0 +1,55 @@
+@e2e
+Feature: Stake Pools Loading
+
+  Background:
+    Given I have completed the basic setup
+    Given I am on the Delegation Centre staking page
+    And I have a "Test Wallet" wallet with funds
+
+  Scenario: "Loading stake pools" message is shown during initial loading of stake pool data
+    Then I should see the following loading message:
+      | message                               |
+      | staking.stakePools.loadingStakePoolsMessage |
+
+  Scenario: Stake pools load error handling works as expected
+    And I should see the following loading message:
+      | message                               |
+      | staking.stakePools.loadingStakePoolsMessage |
+    And Stake pools loading failed
+    Then I should see loading stake pools error message:
+      | message                               |
+      | staking.stakePools.loadingStakePoolsMessage |
+    And I should not see any stake pool
+
+  Scenario: Stake pools are correctly ordered by rank
+    Then I should see the following loading message:
+      | message                               |
+      | staking.stakePools.loadingStakePoolsMessage |
+    And I should see "3" stake pools loaded by rank
+
+  Scenario: Stake pools search works as expected
+    When I see the stake pools search input field
+    And I enter "ROOT" in search input field
+    Then I should see message "Stake pools. Search results: (3)"
+    And I should see "3" stake pool with slug "ROOT"
+
+  Scenario: Stake pool tooltip is correctly displayed and shows correct data
+    When I see "3" stake pools loaded by rank
```

Comment:
Insted this you can just check if stake pools are loaded. You covered ordering in previous scenario

Reply:
Fixed